code/modules/unit_tests/supply_pack.dm

```diff
@@ -0,0 +1,12 @@
+/datum/unit_test/supply_pack/Run()
+	for(var/pack_type in subtypesof(/datum/supply_pack))
+		var/datum/supply_pack/pack = new pack_type
+		var/created = pack.generate(run_loc_bottom_left)
+
+		var/datum/export_report/rep = export_item_and_contents(created, allowed_categories = ALL, apply_elastic = FALSE)
+		var/value = 0
+		for(var/thing in rep.total_value)
+			value += rep.total_value[thing]
+
+		if(value >= pack.cost)
+			Fail("[pack] ([pack_type]) was resold for [value], [value - pack.cost] greater than the buy price of [pack.cost]!")
```

Comment:
This also causes a Fail when the resell value is the same as the purchase cost. But the fail reason will report that as '0 greater than the buy price'